frontend/jest.config.js

```diff
@@ -10,31 +15,21 @@ module.exports = {
     '!**coverage/**',
     '!**lib/**',
   ],
-  moduleNameMapper: {
-    // Handle CSS imports (with CSS modules)
-    // https://jestjs.io/docs/webpack#mocking-css-modules
-    '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
-
-    // Handle CSS imports (without CSS modules)
-    '^.+\\.(css|sass|scss)$': '<rootDir>/src/__mocks__/styleMock.js',
 
-    // Handle image imports
-    // https://jestjs.io/docs/webpack#handling-static-assets
-    '^.+\\.(jpg|jpeg|png|gif|webp|avif|svg)$': `<rootDir>/src/__mocks__/fileMock.js`,
-
-    // Handle module aliases
+  moduleNameMapper: {
     '^@/components/(.*)$': '<rootDir>/components/$1',
+    '\\.svg': '<rootDir>/src/tests/__mocks__/svg.js',
+    '^@/pages/(.*)$': '<rootDir>/pages/$1',
   },
   setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
   testPathIgnorePatterns: ['<rootDir>/node_modules/'],
-  testEnvironment: 'jsdom',
+  testEnvironment: 'jest-environment-jsdom',
```

Comment:
Not sure     if this is correct, lets revisit this

Reply:
I think it's ok - we took this from next.js testing example - https://github.com/vercel/next.js/blob/canary/examples/with-jest/jest.config.js